### PR TITLE
release 2.0 changes

### PR DESCRIPTION
**Pull request by SeanFeldman**

### CODE REVIEW

**Comment by SeanFeldman** [comment]
@Particular/azure-maintainers removing `release` branch in favor of working with `develop` only.

File: src/NServiceBus.Persistence.AzureStorage/NServiceBus.Persistence.AzureStorage.csproj [hunk 1]
```diff
@@ -5,10 +5,10 @@
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>$(SolutionDir)NServiceBus.snk</AssemblyOriginatorKeyFile>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <!-- <GeneratePackageOnBuild>true</GeneratePackageOnBuild> -->
```
Comment (boblangley) [inline src/NServiceBus.Persistence.AzureStorage/NServiceBus.Persistence.AzureStorage.csproj]: Why is this left here commented out?

Comment (SeanFeldman) [reply src/NServiceBus.Persistence.AzureStorage/NServiceBus.Persistence.AzureStorage.csproj]: It was removed. I commented it out instead.

Comment (SeanFeldman) [reply src/NServiceBus.Persistence.AzureStorage/NServiceBus.Persistence.AzureStorage.csproj]: Discussed. Reverting.

**Pull request merged**